src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -406,3 +406,21 @@ td.main, td.menu {
RoIGlkPSJTVkdJRF8yXyI+CgkJCTx1c2UgeGxpbms6aHJlZj0iI1NWR0lEXzFfIiAgb3ZlcmZsb3c9InZpc2libGUiLz4KCQk8L2NsaXBQYXRoPgoJCQoJCQk8aW1hZ2Ugb3ZlcmZsb3c9InZpc2libGUiIGNsaXAtcGF0aD0idXJsKCNTVkdJRF8yXykiIHdpZHRoPSIxMjgiIGhlaWdodD0iMTI4IiB4bGluazpocmVmPSJkYXRhOmltYWdlL2pwZWc7YmFzZTY0LC85ai80QUFRU2taSlJnQUJBZ0VBU0FCSUFBRC83QUFSUkhWamEza0FBUUFFQUFBQUhnQUEvKzRBSVVGa2IySmxBR1RBQUFBQUFRTUEKRUFNQ0F3WUFBQUhGQUFBQjJ3QUFBZjMvMndDRUFCQUxDd3NNQ3hBTURCQVhEdzBQRnhzVUVCQVVHeDhYRnhjWEZ4OGVGeG9hR2hvWApIaDRqSlNjbEl4NHZMek16THk5QVFFQkFRRUJBUUVCQVFFQkFRRUFCRVE4UEVSTVJGUklTRlJRUkZCRVVHaFFXRmhRYUpob2FIQm9hCkpqQWpIaDRlSGlNd0t5NG5KeWN1S3pVMU1EQTFOVUJBUDBCQVFFQkFRRUJBUUVCQVFQL0NBQkVJQUlBQWdBTUJJZ0FDRVFFREVRSC8KeEFCbEFBRUJBUUVCQUFBQUFBQUFBQUFBQUFBQUFRY0dBZ0VCQUFBQUFBQUFBQUFBQUFBQUFBQUFBQkFCQUFNQkFRQUFBQUFBQUFBQQpBQUFBQVFBd1lGQVJFUUVBQUFBQUFBQUFBQUFBQUFBQUFBQ0FFZ0VBQUFBQUFBQUFBQUFBQUFBQUFBQmcvOW9BREFNQkFBSVJBeEVBCkFBRFB3QUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFEUlRPbWlqT21pUXp4b1VNK2FCRGdBQUFORnpyUlJFRVFSQktPQUFBQTBUTzkKREVRU0FBRGdBQUFOQ3ozUVI1QkZBRkRQd0FBTy93Q0E3OFJRQlFVUC85b0FDQUVDQUFFRkFNQi8vOW9BQ0FFREFBRUZBTUIvLzlvQQpDQUVCQUFFRkFONHR5eGJsdldlM0xGdlhzLy9hQUFnQkFnSUdQd0FCLzlvQUNBRURBZ1kvQUFILzJnQUlBUUVCQmo4QUFILy8yUT09Ij4KCQk8L2ltYWdlPgoJPC9nPgoJPGc+CgkJPGRlZnM+CgkJCTxwYXRoIGlkPSJTVkdJRF8zXyIgZD0iTTU0LDc1LjEyYzEzLjg1MiwwLDI1LjEyLTExLjI2OSwyNS4xMi0yNS4xMmMwLTEzLjg1Mi0xMS4yNjktMjUuMTItMjUuMTItMjUuMTJTMjguODgsMzYuMTQ5LDI4Ljg4LDUwCgkJCQlDMjguODgsNjMuODUyLDQwLjE0OCw3NS4xMiw1NCw3NS4xMiBNNTQsODhjLTIwLjk4NywwLTM4LTE3LjAxMy0zOC0zOFMzMy4wMTMsMTIsNTQsMTJDNzQuOTg2LDEyLDkxLjk5OSwyOS4wMTQsOTEuOTk5LDUwCgkJCQlTNzQuOTg2LDg4LDU0LDg4TDU0LDg4TDU0LDg4eiIvPgoJCTwvZGVmcz4KCQk8Y2xpcFBhdGggaWQ9IlNWR0lEXzRfIj4KCQkJPHVzZSB4bGluazpocmVmPSIjU1ZHSURfM18iICBvdmVyZmxvdz0idmlzaWJsZSIvPgoJCTwvY2xpcFBhdGg+CgkJCgkJCTxpbWFnZSBvdmVyZmxvdz0idmlzaWJsZSIgY2xpcC1wYXRoPSJ1cmwoI1NWR0lEXzRfKSIgd2lkdGg9IjEyOCIgaGVpZ2h0PSIxMjgiIHhsaW5rOmhyZWY9ImRhdGE6aW1hZ2UvanBlZztiYXNlNjQsLzlqLzRBQVFTa1pKUmdBQkFnRUFTQUJJQUFELzdBQVJSSFZqYTNrQUFRQUVBQUFBSGdBQS8rNEFJVUZrYjJKbEFHVEFBQUFBQVFNQQpFQU1DQXdZQUFBSGVBQUFDR3dBQUFqMy8yd0NFQUJBTEN3c01DeEFNREJBWER3MFBGeHNVRUJBVUd4OFhGeGNYRng4ZUZ4b2FHaG9YCkhoNGpKU2NsSXg0dkx6TXpMeTlBUUVCQVFFQkFRRUJBUUVCQVFFQUJFUThQRVJNUkZSSVNGUlFSRkJFVUdoUVdGaFFhSmhvYUhCb2EKSmpBakhoNGVIaU13S3k0bkp5Y3VLelUxTURBMU5VQkFQMEJBUUVCQVFFQkFRRUJBUVAvQ0FCRUlBSUFBZ0FNQklnQUNFUUVERVFILwp4QUJrQUFFQkFRRUJBQUFBQUFBQUFBQUFBQUFBQWdjREJnRUJBQUFBQUFBQUFBQUFBQUFBQUFBQUFCQUJBUUVCQVFBQUFBQUFBQUFBCkFBQUFBQkVTSUdBUkFRQUFBQUFBQUFBQUFBQUFBQUFBQUlBU0FRQUFBQUFBQUFBQUFBQUFBQUFBQUdELzJnQU1Bd0VBQWhFREVRQUEKQU0vMUxMZFNBQUFBQUdXNmxsb0ExTExkU0FBQUFBR1c2bGxvQTFMTGRRS1NLU0tTS1NLU0t5M1VNdkFHblpqcFphQmFCYUJhQmFCZQpZNlhtZ0EwZk9OQ09ya09ya09ya09ya09ya091Y2FGbm9BOTk0SDNoU1JTUlNSU1JTUlhnZmVlREFBQUFBQUFBQUFBQUFBQUFBUC9hCkFBZ0JBZ0FCQlFEd0gvL2FBQWdCQXdBQkJRRHdILy9hQUFnQkFRQUJCUUR5TlZWVlZWVlZlYXFxcXFxcXE4NmFhYWFhYWFhYWFhNTAKMDAwMDAwMDAwMDF6cHBwcHBwcHBwcHBybXFxcXFxcXF2bFAvMmdBSUFRSUNCajhBQWYvYUFBZ0JBd0lHUHdBQi85b0FDQUVCQVFZLwpBQUIvLzlrPSI+CgkJPC9pbWFnZT4KCTwvZz4KPC9nPgo8L3N2Zz4=);
```

Comment:
it looks to me that what you are actually expecting was ``background-image`` btw, according on your usage of ``background-repeat: no-repeat;``